### ISSUE TITLE
Custom scoring doesn't work only when 'n_jobs = -1'

### ISSUE THREAD

**Issue by mariomm17** [EpistasisLab/tpot#970]
When I create my own score function and feed it into TPOT, I can't create a model when I set  **n_jobs = -1**.  My new score function is :

![image](https://user-images.githubusercontent.com/47488921/70043220-5c05a780-15c0-11ea-8188-5e5e51a02bdf.png)

As well, when I execute the following:

![image](https://user-images.githubusercontent.com/47488921/70043980-b7846500-15c1-11ea-84bd-4c591f53fc6d.png)

I obtain the next error:

![image](https://user-images.githubusercontent.com/47488921/70044085-ed294e00-15c1-11ea-81cd-4d11817650a6.png)

I've been reading some errors related to mine, but I couldn't find any solution to this one.
I would appreciate any help!

Thank you,
Mario

**Comment by weixuanfu** [comment]
I think this issue should be fixed in the latest version (0.11) of TPOT. Could you please try the small demo below to check if it works with TPOT v0.11 in your environment? If not, you may also need update your scikit-learn version to > 0.21.

```python
from tpot import TPOTClassifier
from sklearn.datasets import load_digits
from sklearn.model_selection import train_test_split
from sklearn.metrics.scorer import make_scorer

digits = load_digits()
X_train, X_test, y_train, y_test = train_test_split(digits.data, digits.target,
                                                    train_size=0.75, test_size=0.25, random_state=42)
# Make a custom metric function
def my_custom_accuracy(y_true, y_pred):
    return float(sum(y_pred == y_true)) / len(y_true)

# Make a custom a scorer from the custom metric function
# Note: greater_is_better=False in make_scorer below would mean that the scoring function should be minimized.
my_custom_scorer = make_scorer(my_custom_accuracy, greater_is_better=True)

tpot = TPOTClassifier(generations=5, population_size=20, verbosity=2,
                      scoring=my_custom_scorer, n_jobs=-1, random_state=42)
tpot.fit(X_train, y_train)
```

**Comment by mariomm17** [comment]
Thanks for your help, problem solved!

Also, thanks for your quick answer.

Mario
